lib/model/folder_sendrecv.go

```diff
@@ -315,18 +315,23 @@ func (f *sendReceiveFolder) processNeeded(dbUpdateChan chan<- dbUpdateJob, copyC
 			return true
 		}
 
+		if runtime.GOOS == "windows" && fs.WindowsInvalidFilename(intf.FileName()) {
```

Comment:
But now we can no longer escape complaints and errors on invalid file names by ignoring them